### ISSUE TITLE
Change boolean props to be false by default

### ISSUE THREAD

**Issue by targos** [zakodium/react-plot#88]
After working a bit with it, I think having `true` by default for some props is confusing. I propose that if we have a boolean prop, it is  always `false` by default. It means that for example we should change `<Axis display={true} />` to be `<Axis noDisplay={true} />`.

**Comment by maasencioh** [comment]
I think that last time we agree on it, but we didn't create an issue for it

**Comment by targos** [comment]
@lpatiny @stropitek @Sebastien-Ahkrin what do you think?

**Comment by stropitek** [comment]
I agree!

**Comment by stropitek** [comment]
I would find it confusing to go from `<Axis />` to `<Axis display />` and see no difference in the result

**Comment by targos** [comment]
Of course, instead of `noDisplay`, it could be named `hidden`. If this is widely accepted, we should go through all boolean props and find a good naming convention.

**Comment by maasencioh** [comment]
![approved](https://media.giphy.com/media/13zeE9qQNC5IKk/giphy.gif)
